components/camel-tracing/src/main/java/org/apache/camel/tracing/decorators/AbstractSpanDecorator.java

```diff
@@ -99,9 +100,14 @@ public void pre(SpanAdapter span, Exchange exchange, Endpoint endpoint) {
         span.setComponent(CAMEL_COMPONENT + scheme);
 
         // Including the endpoint URI provides access to any options that may
-        // have been provided, for
-        // subsequent analysis
-        span.setTag("camel.uri", URISupport.sanitizeUri(endpoint.getEndpointUri()));
+        // have been provided, for subsequent analysis
+        String uri = URISupport.sanitizeUri(endpoint.getEndpointUri());
+        span.setTag("camel.uri", uri);
+        span.setTag(TagConstants.URL_PATH, stripSchemeAndOptions(endpoint));
```

Comment:
Should URL_SCHEME also be set as a tag ?

Reply:
Yes I think so. I've added it.